docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_PanelChanger.htm

```diff
@@ -14,6 +14,7 @@ <h1>Panel Changer</h1>
 <p>Panel can have header, content and so on like the page component.<br>
 It works even if the panel components is in other HTML files.<br>
 Panel changer component controls lifecycle of the panels.</p>
+<img src="../../images/mobile_panelchanger.png" alt="photo of panelchanger element in Tizen">
 
 <button
```

Comment:
Maybe a line break is needed here.

The "Use example" button is right next to the image.

![image](https://user-images.githubusercontent.com/9958374/71229620-9e9de280-2329-11ea-8a49-034c58e4bf85.png)

Reply:
Done